src/pages/guides/sellers/image-tips.md

```diff
@@ -4,9 +4,9 @@ title: Image tips
 
 # Image tips
 
-An image makes a powerful and lasting impression without a word. Images uploaded to your account help people learn about your products, and also represent your brand. Use only high-quality images for your personal, company, and extension profiles. The Magento logo appears as a placeholder in your personal, company, and extension profiles until you upload your images of your own.
+An image makes a powerful and lasting impression without a word. Images uploaded to your account help people learn about your products, and also represent your brand. Use only high-quality images for your personal, company, and listing profiles. The Magento logo appears as a placeholder until you upload your images of your own.
```

Comment:
Please, remove sentence "The Magento logo appears as a placeholder until you upload your images of your own". 
I test it and there is no magento logo while there are no images uploaded. 

![image](https://user-images.githubusercontent.com/44397551/236833102-002e366b-5d77-4d0d-b92b-d344545cbf9a.png)